lib/reform/form/active_model/validations.rb

```diff
@@ -143,7 +143,8 @@ def to_s
 
           def add(key, error_text)
             # use rails magic to get the correct error_text and make sure we still update details and fields
-            text = @amv_errors.add(key, error_text)
+            error = @amv_errors.add(key, error_text)
+            error = [error.message] if error.respond_to?(:message)
```

Comment:
What's happening when `error` doesn't have a `message` method? Is that Rails < 6.1? I'd refrain from introducing `#respond_to` in TRB gems. https://youtu.be/mjsnd8dJbew?t=1113

Reply:
Unfortunately, in the case of Rails < 6.1, `error` doesn't have a `message` method and causes an error. 

```
NoMethodError: undefined method `message' for ["error_text"]:Array
    lib/reform/form/active_model/validations.rb:147:in `add' 
``` 

> I'd refrain from introducing #respond_to in TRB gems 

Oh, I see. So what do you think about checking Rails' version?

Reply:
I updated to use `is_a?` instead of `respond_to?`.